Sources/BrowserServicesKit/Autofill/AutofillUserScript+SecureVault.swift

```diff
@@ -329,6 +330,7 @@ extension AutofillUserScript {
     struct GetAutofillDataRequest: Codable {
         let mainType: GetAutofillDataMainType
         let subType: GetAutofillDataSubType
+        let isAutoprompt: Bool
```

Comment:
@GioSensation could we represent this as an enum? firstly to match how it's represented in the JS side, but also to prevent restricting ourselves to a binary choice here.